src/main/scala/com/fulcrumgenomics/commons/util/DelimitedDataParser.scala

```diff
@@ -73,7 +73,7 @@ class Row private[util] (private val headerIndices: Map[String,Int], private val
   /** Gets a value from the specified column. If the value is null or empty returns None.  If
     * `allowMissingColumn` is true, then None is returned if the given column name is not present. */
```

Comment:
I had to re-read this comment a few times to see what the intended behavior was.  I think will make it clearer, along with your bugfix.
```suggestion
  /** Gets a value from the specified column. If the value is null or empty returns None.  If
    * `allowMissingColumn` is true, then None is also returned if the given column name is not present. */
```